force-app/main/default/classes/MaintenanceRequestHelper.cls

```diff
@@ -0,0 +1,57 @@
+public with sharing class MaintenanceRequestHelper {
+
+    public static void createMaintenanceRequestRecord(List<Case> caseList) {
+        try{
+            List<Case> newCases = new List<Case>();
+            Integer cycle = getMaintenanceCycle(caseList) == null ? 0 : getMaintenanceCycle(caseList);
```

Comment:
better to call once getMaintenanceCycle() and then use returned value where needed